### PR TITLE
Add ability to invite contact by phone number

### PR DESCRIPTION
**Pull request by vp8x8**

### CODE REVIEW

**Comment by vp8x8** [comment]
When you click it, a _phone_ entry is added to tag list just like a regular phone entry. You can remove it and also add a _normal_ (message) invite. This is the standard way of adding stuff this particular AtlasKit component allows us to do.

We can add a name alongside to the phone number but the created tag will be truncated. Do you want me to add one?

**Comment by saghul** [comment]
Can you please look at the test failures? Also, please provide us with a screenshot or 2.

**Comment by vp8x8** [comment]
Screenshot 2020-01-30 at 13 50 19

File: react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js [hunk 1]
```diff
@@ -283,24 +283,42 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
...
      */
     _parseQueryResults(response = []) {
-        const { t } = this.props;
+        const { t, _dialOutEnabled } = this.props;
         const users = response.filter(item => item.type !== 'phone');
-        const userDisplayItems = users.map(user => {
-            return {
-                content: user.name,
-                elemBefore: <Avatar
-                    className = { 'avatar-small' }
```
Comment (damencho) [inline react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js]: We used to have to classes 'avatar-small' and 'avatar-xsmall' which are 28x28 and 16x16, one is used in elemBefore and one int tag: {elemBefore....}, what is the difference? And why you switched to using just the smaller one?
Other than that everything seems fine.
And if we are sure and want to remove one of them and not use it, can you clean up css, please.

Comment (vp8x8) [reply react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js]: @damencho Thanks for the catch! Fixed it. 

One is the avatar for the `tag` (the input where the completions are added) and the other one for the element per se (the list entry). The tag avatar is smaller and the tag for list entry has to be bigger.

**Pull request merged**